[zeroc-ice] Build failure

Package: zeroc-ice[core,glacier2lib,glacier2router,icegridlib,icessl,icestormlib]:x64-windows -> 3.7.9#1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 365bc2f5b 2023-08-22 (2 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks zeroc-ice[core,glacier2router]`

**Failure logs**

```
-- Using cached zeroc-ice-ice-v3.7.9.tar.gz.
-- Extracting source C:/v/downloads/zeroc-ice-ice-v3.7.9.tar.gz
-- Applying patch mcppd_fix.patch
-- Applying patch no-werror.patch
-- Using source at C:/v/vcpkg3/buildtrees/zeroc-ice/src/v3.7.9-353afb192c.clean
-- Modifying hardcoded 'RuntimeLibrary' directives in .vcxproj files
-- Making Ice Builder for MSBuild available
-- Using cached zeroc.icebuilder.msbuild.5.0.9.zip.
-- Extracting source C:/v/downloads/zeroc.icebuilder.msbuild.5.0.9.zip
-- Using source at C:/v/vcpkg3/buildtrees/zeroc-ice/src/icebuilder-8509f8b6da.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for x64-windows
-- Building cpp/msbuild/ice.v143.sln for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: msbuild C:/v/vcpkg3/buildtrees/zeroc-ice/x64-windows-rel/cpp/msbuild/ice.v143.sln /p:Configuration=Release /p:UseVcpkg=yes /p:IceBuildingSrc=yes /t:C++11\\icessl++11 /t:C++11\\glacier2++11 /t:C++98\\glacier2router /t:C++98\\glacier2cryptpermissionsverifier /t:C++11\\icegrid++11 /t:C++11\\icestorm++11 /p:ForceImportAfterCppProps=C:/v/vcpkg3/buildtrees/zeroc-ice/x64-windows/vcpkg_msbuild.props /p:ForceImportAfterCppTargets=C:/v/vcpkg3/buildtrees/zeroc-ice/x64-windows/vcpkg_msbuild.targets /t:C++11\\ice++11 /p:UseMultiToolTask=true /p:MultiProcMaxCount=21 /p:EnforceProcessCountAcrossBuilds=true /m:21 -maxCpuCount:21 /p:Platform=x64 /p:PlatformTarget=x64 /p:PlatformToolset=v143 /p:WindowsTargetPlatformVersion=10.0.22621.0 /p:VcpkgApplocalDeps=false /p:VcpkgManifestInstall=false /p:VcpkgManifestEnabled=false /p:VcpkgEnabled=false /p:VcpkgTriplet=x64-windows /p:VcpkgInstalledDir=C:/v/vcpkg3/installed
    Working Directory: C:/v/vcpkg3/buildtrees/zeroc-ice/x64-windows-rel
    Error code: 1
    See logs for more information:
      C:\v\vcpkg3\buildtrees\zeroc-ice\build-x64-windows-rel-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-msbuild/vcpkg_msbuild_install.cmake:108 (vcpkg_execute_required_process)
  ports/zeroc-ice/portfile.cmake:268 (vcpkg_msbuild_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\v\vcpkg3\buildtrees\zeroc-ice\build-x64-windows-rel-out.log</summary>

```
MSBuild version 17.7.2+d6990bcfa for .NET Framework
Build started 22/08/2023 22:06:09.

     1>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\ice.v143.sln" on node 1 (C++11\icessl++11;C++11\glacier2++11;C++98\glacier2router;C++98\glacier2cryptpermissionsverifier;C++11\icegrid++11;C++11\icestorm++11;C++11\ice++11 target(s)).
     1>ValidateSolutionConfiguration:
         Building solution configuration "Release|x64".
     1>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\ice.v143.sln" (1) is building "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\IceSSL\msbuild\icessl++11\icessl++11.vcxproj.metaproj" (2) on node 1 (default targets).
     2>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\IceSSL\msbuild\icessl++11\icessl++11.vcxproj.metaproj" (2) is building "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice++11\ice++11.vcxproj.metaproj" (3) on node 1 (default targets).
     3>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice++11\ice++11.vcxproj.metaproj" (3) is building "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\slice2cpp\msbuild\slice2cpp.vcxproj.metaproj" (4) on node 1 (default targets).
     4>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\slice2cpp\msbuild\slice2cpp.vcxproj.metaproj" (4) is building "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Slice\msbuild\slice.vcxproj.metaproj" (5) on node 1 (default targets).
     5>Project "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Slice\msbuild\slice.vcxproj.metaproj" (5) is building "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\IceUtil\msbuild\iceutil\iceutil.vcxproj" (6) on node 1 (default targets).
     6>PrepareForBuild:
         Creating directory "x64\Release\".
         Creating directory "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\..\..\cpp\lib\x64\Release\".
         Creating directory "x64\Release\iceutil.tlog\".
       InitializeBuildStatus:
         Creating "x64\Release\iceutil.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
         Touching "x64\Release\iceutil.tlog\unsuccessfulbuild".
       ClCompile:
         C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\bin\HostX64\x64\CL.exe /c /I"C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\..\..\cpp\include\generated\cpp98\x64\Release" /I"C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\..\..\cpp\include" /I"C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\..\..\cpp\src" /I"C:/v/vcpkg3/installed/x64-windows/include" /I"C:/v/vcpkg3/packages/zeroc-ice_x64-windows/include" /I"C:/v/vcpkg3/installed/x64-windows/include" /Zi /nologo /W4 /WX /diagnostics:column /FS /O2 /Oy- /D ICE_BUILDING_SLICE_COMPILERS /D ICE_BUILDING_SRC /D NDEBUG /D ICE_STATIC_LIBS /D _CONSOLE /D _WIN32_WINNT=0x601 /D WIN32_LEAN_AND_MEAN /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"x64\Release\\" /Fd"x64\Release\iceutil-objs.pdb" /external:W4 /Gd /TP /wd4121 /wd4250 /wd4251 /wd4275 /wd4324 /FC /errorReport:queue /bigobj    -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -GR -EHsc -MP  -MD -O2 -Oi -Gy -DNDEBUG -Z7    /WX- ..\..\ConsoleUtil.cpp
...
Skipped 2604 lines
...

         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2Lib\msbuild\glacier2++11\glacier2++11.vcxproj]


       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\ice.v143.sln" (C++11\icessl++11;C++11\glacier2++11;C++98\glacier2router;C++98\glacier2cryptpermissionsverifier;C++11\icegrid++11;C++11\icestorm++11;C++11\ice++11 target) (1) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2\msbuild\glacier2router.vcxproj.metaproj" (default target) (13) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\IceSSL\msbuild\icessl\icessl.vcxproj.metaproj" (default target) (14) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj.metaproj" (default target) (15) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj" (default target) (17) ->
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : command line  warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : command line  warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : command line  warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/W4' with '/W3' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/Zi' with '/Z7' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         cl : Command line warning D9025: overriding '/WX' with '/WX-' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]


       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\msbuild\ice.v143.sln" (C++11\icessl++11;C++11\glacier2++11;C++98\glacier2router;C++98\glacier2cryptpermissionsverifier;C++11\icegrid++11;C++11\icestorm++11;C++11\ice++11 target) (1) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Glacier2\msbuild\glacier2router.vcxproj.metaproj" (default target) (13) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\IceSSL\msbuild\icessl\icessl.vcxproj.metaproj" (default target) (14) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj.metaproj" (default target) (15) ->
       "C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj" (default target) (17) ->
       (ClCompile target) -> 
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2039: 'binary_function': is not a member of 'std' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2504: 'binary_function': base class undefined [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,57): error C2143: syntax error: missing ',' before '<' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2039: 'binary_function': is not a member of 'std' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2504: 'binary_function': base class undefined [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,57): error C2143: syntax error: missing ',' before '<' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2039: 'binary_function': is not a member of 'std' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2504: 'binary_function': base class undefined [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,57): error C2143: syntax error: missing ',' before '<' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2039: 'binary_function': is not a member of 'std' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,42): error C2504: 'binary_function': base class undefined [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]
         C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\include\IceUtil\Timer.h(105,57): error C2143: syntax error: missing ',' before '<' [C:\v\vcpkg3\buildtrees\zeroc-ice\x64-windows-rel\cpp\src\Ice\msbuild\ice\ice.vcxproj]

    879 Warning(s)
    12 Error(s)

Time Elapsed 00:02:08.02
```
</details>

**Additional context**

--- a/ports/zeroc-ice/fix-missing-functional.patch
+++ b/ports/zeroc-ice/fix-missing-functional.patch
@@ -1,0 +1,12 @@
+diff --git a/cpp/include/IceUtil/Timer.h b/cpp/include/IceUtil/Timer.h
+index 177a7f8..19b8c09 100644
+--- a/cpp/include/IceUtil/Timer.h
++++ b/cpp/include/IceUtil/Timer.h
+@@ -12,6 +12,7 @@
+ 
+ #include <set>
+ #include <map>
++#include <functional>
+ 
+ namespace IceUtil
+ {

--- a/ports/zeroc-ice/portfile.cmake
+++ b/ports/zeroc-ice/portfile.cmake
@@ -4,7 +4,10 @@ vcpkg_from_github(
     REPO zeroc-ice/ice
     REF "v${VERSION}"
     SHA512 07d7c439fbe1f69d808d05a11f32e09cdd8d4df2a93b6f253496304e0a521d417212ae688e316b4450dae406b59d1a460025b51ecd0614c69e48d86c0a6f81c5
-    PATCHES mcppd_fix.patch no-werror.patch
+    PATCHES
+        mcppd_fix.patch
+        no-werror.patch
+        fix-missing-functional.patch
 )
 
 set(RELEASE_TRIPLET ${TARGET_TRIPLET}-rel)

--- a/ports/zeroc-ice/vcpkg.json
+++ b/ports/zeroc-ice/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "zeroc-ice",
   "version": "3.7.9",
-  "port-version": 1,
+  "port-version": 2,
   "maintainers": "Benjamin Oldenburg <benjamin.oldenburg@ordis.co.th>",
   "description": "Comprehensive RPC framework with support for C++, CSharp, Java, JavaScript, Python and more.",
   "homepage": "https://github.com/zeroc-ice/ice",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -9038,7 +9038,7 @@
     },
     "zeroc-ice": {
       "baseline": "3.7.9",
-      "port-version": 1
+      "port-version": 2
     },
     "zeromq": {
       "baseline": "2023-06-20",

--- a/versions/z-/zeroc-ice.json
+++ b/versions/z-/zeroc-ice.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "dc07eee81cdf5c66efc83d606c635e722ef23d39",
+      "version": "3.7.9",
+      "port-version": 2
+    },
+    {
       "git-tree": "bb449d6446b3700170a831c0b155d58be3613a26",
       "version": "3.7.9",
       "port-version": 1

